deepchecks/nlp/input_validations.py

```diff
@@ -0,0 +1,110 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2023 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing input validation functions."""
+from typing import Dict, Optional, Sequence
+
+import numpy as np
+import pandas as pd
+
+from deepchecks.core.errors import DeepchecksValueError
+from deepchecks.nlp.task_type import TaskType, TTextLabel
+from deepchecks.utils.logger import get_logger
+from deepchecks.utils.type_inference import infer_categorical_features
+from deepchecks.utils.validation import is_sequence_not_str
+
+
+def validate_tokenized_text(tokenized_text: Optional[Sequence[Sequence[str]]]):
+    """Validate tokenized text format."""
+    error_string = 'tokenized_text must be a Sequence of sequences of strings'
```

Comment:
```suggestion
    error_string = 'tokenized_text must be a Sequence of Sequences of strings'
```
Or without capitals in both.